komorebi/settings.py

```diff
@@ -0,0 +1,149 @@
+import logging
+import os
+from enum import Enum
+
+from gi.repository import Gio, GLib
+
+import komorebi.consts
+
+key_file_group : str = 'KomorebiProperties'
+
+# Settings
+wallpaper_name : str = 'foggy_sunny_mountain'
```

Comment:
Shouldn't this be inside a `@dataclass` `class`?

Reply:
Fixed; it's not perfect still, but these variables are now inside a `Settings` class.